src/wasm.rs

```diff
@@ -66,7 +66,7 @@ fn format_kv_pairs(record: &Record) -> Option<Object> {
             if self.hashmap.is_none() {
                 self.hashmap = Some(HashMap::new())
             }
-            let hm = self.hashmap.as_mut().unwrap();
+            let hm = self.hashmap.as_mut()?;
```

Comment:
ah, I know remember why we had to have `unwrap` here - wasm breaks with `?` - and I wasn't not entirely sure why then, and still not sure why now. 

I had this merged and published, but I'll have to revert from master tomorrow since this broke `async-std`.

Reply:
That's weird. We publish a lot of other stuff in Surf / Tide / http-rs that uses `?` and is (seemingly) fine...?

@yoshuawuyts?